Commonalities/documentation/API-design-guidelines.md

```diff
@@ -1089,3 +1090,259 @@ following aspects:
 - Security Flow description applied (String)
 - Endpoint token URL
 - URL to endpoint authorization ( If flow is based on "`authorizationCode`").	
+
+## 12. Subscription, Notification & Event
+
+In order to provide event-based interaction, CAMARA API could provided capabilities for subscription & notification management.
+A subscription allows an API consumer to request event notification reception at a given URL (callback address) for a specific context.
+A notification is the publication at the listener address about an occurred event.
+Managed event types are explicitly defined in CAMARA API OAS.
+
+### 12.1 Subscription
+
+We distinguish 2 types of subscriptions:
+- Instance-based subscription (indirect creation)
+- Resource-based subscription (direct creation)
+
+**Instance-based subscription**
+
+An instance-based subscription is a subscription indirectly created, additionally to another resource creation. For example for a Payment request (in Carrier Billing API), in the `POST/payments`, the API consumer could request to get event notification about **this** Payment request processing update. The subscription is not an autonomous entity and its lifecycle is linked to the managed entity (the Payment resource in this case). The subscription terminates with the managed entity.
```

Comment:
Would it be relevant to offer a generic example related to the TELCO if it is applicable,  like  an instance-based subscription could be used to notify a customer when the bandwidth usage of a specific virtual network function (VNF) reaches a certain threshold or it can be used to notify a customer when the topology of their network changes, such as when a new VNF is added or removed other than payment/billing example

Reply:
@sachinvodafone I tried to provide in the document a example using Roaming notification which is related to telco and important related to one of our API. For my perspective it is enough and after it will be up to each API workgroup to provide good example in each documentation.